llmfoundry/utils/exceptions.py

```diff
@@ -532,3 +534,18 @@ class UCNotEnabledError(UserError):
     def __init__(self) -> None:
         message = f'Unity Catalog is not enabled on your cluster.'
         super().__init__(message)
+
+
+class DeltaTableNotFoundError(UserError):
+    """Error thrown when the delta table passed in training does not exist or is not found."""
+
+    def __init__(self, message: str) -> None:
+        self.message = message
+        super().__init__(message)
```

Comment:
Can you pass me the table name so I can construct a good message for it in mcloud? or we could pass this message directly to the user if you feel it is perfect for the user.

Reply:
The current message is:

```
Error processing `catalog`.`volume_name`.`table_name`: [DELTA_TABLE_NOT_FOUND] Delta table `volume_name`.`table_name` doesn't exist.
```

which I think is pretty good.

Reply:
I think @jjanezhang wanted the table name as an attr on the error message to use downstream

Reply:
Yea I think I'm going to rewrite to be something like "Your {train/eval} data path {table name} does not exist. Please double check your delta table name". If you could pass the delta table name we check as an attr on the class that would be great!